app/controllers/articles_controller.rb

```diff
@@ -84,6 +84,16 @@ def destroy
     redirect_to :articles
   end
 
+  def destroy_all
+    @article = current_user.articles
+    if @article&.destroy_all
+      flash[:success] = 'Your reset list has been reset.'
```

Comment:
Looks like there's a small typo in that sentence ("reset list"). We could use this wording:

```suggestion
      flash[:success] = 'Your reading list is now empty. 🍃'
```